src/ui/interface.js

```diff
@@ -228,4 +230,67 @@ module.exports = function Interface(options) {
     $("[rel=popover]").popover()
     return true;
   });
+
+  //Start Handle multiple webcam resolutions
+  function changeResolution(w, h){
+    document.getElementById('image').setAttribute("width",w);
+    document.getElementById('image').setAttribute("height",h);
+  }
+  $('#qvga').click(function(e){
+    changeResolution('100px','100px')
+  });
+  $('#vga').click(function(e){
+    changeResolution('800px','600px')
+  });
+  $('#hd').click(function(e){
+    changeResolution('2400px','1800px')
```

Comment:
Ah, here, should we stick to a standard definition of HD? This table was kind of helpful... we might say "720 HD" or "1080 HD"?

https://en.wikipedia.org/wiki/List_of_common_resolutions#Digital_Standards

Reply:
I see you addressed this! 👍🏽

Reply:
Done as requested